ckanext/ontario_theme/templates/snippets/add_dataset.html

```diff
@@ -0,0 +1,10 @@
+{# Adds 'Add Dataset' button #}
```

Comment:
Unsure where this file is used within this PR and set of changes.

Reply:
just wanted to note that. I'm accepting for now and we can undo changes in future.